### ISSUE TITLE
Workflow: update directly related modules fields

### ISSUE THREAD

**Issue by joebordes** [tsolucio/corebos#206]
Enhance workflow update field task to support updating fields on directly related modules

**Comment by loridacito** [comment]
Hi Joe, 
I made a pull request on this:
https://github.com/tsolucio/corebos/pull/223

I have a couple of doubts I want to talk to you about. For example now it can be used only choosing fields of the same related module on one update entity task

**Comment by loridacito** [comment]
Tested it with a multiple combination of fields. It works correctly
![screenshot from 2018-01-18 14-43-53](https://user-images.githubusercontent.com/9430573/35101059-6af38424-fc5e-11e7-86d8-ab0b6d97f705.png)

**Comment by loridacito** [comment]
When more than 3 related modules should be updated , the task is added in the queue (as if it is a scheduled task) and executed after 5 minutes

**Comment by joebordes** [comment]
Spectacular!! This change is wonderful. It has been on my to-do list for way too long. Thank you!

I applied the patch, did some further cleanup and started testing. I REALLY like how you solved the UI part of this problem.

I found two bugs, one I fixed and the other I leave for you (as you probably have a more clear idea of what is happening), one "nice to have" and some additional work to be done, which we can schedule for the future when we have time.

I will create new issues for each as I consider this part of the work done.

Thanks!
